src/en/charms-storage.md

```diff
@@ -198,6 +197,10 @@ a different number. In any case, if detaching storage from a unit would bring
 the total number of storage instances below the minimum, Juju will return an
 error.
 
+It is not possible to add storage to a model without also attaching it to a
```

Comment:
maybe "add *new* storage"? the following sentence ("... you can add storage to a model...") reads like it contradicts this